templates/html-css-static/README.md

```diff
@@ -0,0 +1,40 @@
+# TITLE
+
+<!--
+
+Describe the exercise in a few sentences. E.g.:
+
+This challenge offers a simple static website. Let's make it more gorgeous with HTML and CSS!
+
+-->
+
+## Task
+
+<!--
+
+Explaining the task in detail. E.g.:
+
+Build a personal website according to the ![wireframe](). Use the following hints as guidelines:
+
+- Hint 1
+- Hint 2
+- ...
+
+Switch to the [index.html](./index.html) file and make something great happen!
+
+-->
+
+## Notes
+
+- You only have to touch the `index.html` file.
```

Comment:
Most exercises based on  this template are concerned with teaching CSS and changes are to made in the `styles.css` file.

I think we should change this in template